### PR TITLE
feat(Zerion): add presence

### PR DESCRIPTION
**Pull request by shiseni**
## Description 
Add presence for [Zerion](https://zerion.cc/)
* Resolves: #7163

Settings:
1. Multilanguage,
2. Privacy,
3. Show covers,
4. Show timestamps,
5. Show buttons.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

| Home | Viewing serial page | Viewing Serial | Searching |
| --- | --- | --- | --- |
| image | image | image | image |

</details>

### CODE REVIEW

File: websites/Z/Zerion/presence.ts [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+			if (genres && !privacy) {
+				presenceData.details = strings.viewGenre;
+				presenceData.state = genres;
+			}
+			presenceData.smallImageKey = Assets.Searching;
+			presenceData.smallImageText = strings.search;
+			break;
+
+		case "serial":
+			presenceData.details = `${strings.viewPage}`;
```
Comment (DarkVillager) [inline websites/Z/Zerion/presence.ts:90]:
```suggestion
			presenceData.details = strings.viewPage
```

File: websites/Z/Zerion/presence.ts [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+			presenceData.state =
+				textContent("#series-page .info .title-original") ||
+				textContent("#series-page .info .title");
+			presenceData.largeImageKey =
+				document.querySelector<HTMLImageElement>(".poster img")?.src;
+			presenceData.smallImageKey = Assets.Viewing;
+			presenceData.smallImageText = strings.viewing;
+
+			if (path[3]) {
+				presenceData.details = `${strings.viewPage}`;
```
Comment (DarkVillager) [inline websites/Z/Zerion/presence.ts:100]:
```suggestion
			presenceData.details = strings.viewPage
```

File: websites/Z/Zerion/presence.ts (hunk 1 above)
Comment (shiseni) [reply websites/Z/Zerion/presence.ts]: Forgot to remove. Wanted to insert `${strings.viewPage} ${strings.anime}` but it doesn't look good.

File: websites/Z/Zerion/presence.ts (hunk 2 above)
Comment (shiseni) [reply websites/Z/Zerion/presence.ts]: > 

Same thing.

**Review by matireczunio — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**